axiom/scripts/axiomatic.py

```diff
@@ -121,11 +121,15 @@ def parseOptions(self, args):
             # If a reactor is being selected, it must be done before the store
             # is opened, since that may execute arbitrary application code
             # which may in turn install the default reactor.
-            if "--reactor" in args:
-                reactorIndex = args.index("--reactor")
-                shortName = args[reactorIndex + 1]
-                del args[reactorIndex:reactorIndex + 2]
-                self.opt_reactor(shortName)
+            for index, arg in enumerate(args):
+                if arg.startswith("--reactor"):
+                    if "=" in arg:
+                        shortName = arg.split("=")[1]
```

Comment:
I'm not sure how to actually fix the coverage here, since these lines only run in a subprocess (which coverage obviously has no knowledge of). My reading of the test code agrees with yours (that this branch should be adequately covered by the test you added); maybe @glyph has an idea of how to work around this?

One possibility is of course to avoid running the code in a subprocess, but there's a big comment in the test explaining exactly why it uses a subprocess, so I'm not sure this is a viable route.